releasenotes/notes/add-transactions-errors-COAT-3f6a1ece4f1c324f.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+other:
+  - |
+    Add a new metric counter to the Agent telemetry for the transactions.errors.
+    This will help us to monitor the rate of empty or rejected sketch payloads sent to the backend
```

Comment:
```suggestion
    Add a new metric counter to the Agent telemetry for transactions.errors.
    This will help to monitor the rate of empty or rejected sketch payloads sent to the backend.
```